core/src/main/resources/hudson/model/Job/requirePOST_ru.properties

```diff
@@ -0,0 +1,23 @@
+# The MIT License
+#
+# Copyright (c) 2004-, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number of other of contributors
```

Comment:
Wrong license header.
Please use your own name if you want to have this header in the property file